src/test/java/TestHomework.java

```diff
@@ -1,6 +1,45 @@
-public class TestHomework extends BaseTest{
-
-    public void testMethod() {
-
-    }
-}
+//import org.openqa.selenium.By;
+//import org.openqa.selenium.WebElement;
+//import org.testng.Assert;
+//import org.testng.annotations.Test;
+//
+//public class TestHomework extends BaseTest{
+//
+//
+//
+//    @Test
+//    public void testMethod() throws InterruptedException {
+//        String playlistName = "kz-new playlist";
+//
+//        navigateToPage();
+//        provideEmail("demo@class.com");
+//        providePassword("te$t$tudent");
+//        clickSubmit();
+//        Thread.sleep(2000);
+//
+//        WebElement search = driver.findElement(By.cssSelector("[type='search']"));
+//        search.sendKeys("Pluto");
+//
+//        WebElement viewAll = driver.findElement(By.cssSelector("div.results h1 > button"));
+//        viewAll.click();
+//        Thread.sleep(2000);
+//
+//        WebElement firstSong = driver.findElement(By.cssSelector("section#songResultsWrapper td.title"));
+//        firstSong.click();
+//        Thread.sleep(2000);
+//
+//        WebElement addToButton = driver.findElement(By.cssSelector("section#songResultsWrapper button.btn-add-to"));
+//        addToButton.click();
+//        Thread.sleep(1000);
+//
+//        // search the playlist and click
+//        WebElement playlist = driver.findElement(By.xpath("//section[@id='songResultsWrapper']//li[contains(text(),'kz-new playlist')]"));
+//        playlist.click();
+//
+//        WebElement notif = driver.findElement(By.cssSelector("div.success.show"));
+//        Assert.assertEquals(notif.getText(), "Added 1 song into \"kz-new playlist.\"");
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.